INSTALL.md

```diff
@@ -109,10 +109,23 @@ Also see the README for usage information.
 
 ## Quick version for FreeBSD
 
-The build process for FreeBSD is rather complicated, but the good news is that
-there is a script in the 'scripts' folder that will at least attempt to do all
-the work for you. And should the script not work for you, you can still look
-through it and use it as an installation guide.
+The forked-daapd build process on FreeBSD could be automated via the existing
```

Comment:
It would be good if the instructions are more direct. I'm not sure what is meant by "could be", and also not sure why forking the project would be necessary? If it is sufficient to run the two commands then I think it should just say something like "You can use the ports framework to handle dependencies and build options, so the following is sufficient to build from source on FreeBSD:"

Reply:
The FreeBSD port is good and up-to-date, so the conditional isn't warranted.

Reply:
> It would be good if the instructions were more direct. I'm not sure what is meant by "could be", and also not sure why forking the project would be necessary? If it is sufficient to run the two commands then I think it should just say something like "You can use the ports framework to handle dependencies and build options, so the following is sufficient to build from source on FreeBSD:"

Right, I just wanted to explain to the users that the port can be modified if needed. I'll submit an update shortly.